app/reduxContent/settings/reducers.js

```diff
@@ -82,6 +91,46 @@ export default handleActions(
       }
       return state;
     },
+    [ADD_PATH]: (state, action) => {
+      const newPath = action.path;
+      const pathsList = state.get('pathsList');
+      const indexFound = pathsList.findIndex(item => {
+        return item.get('label') === newPath.label;
+      });
+
+      if (indexFound === -1) {
+        return state.set('pathsList', pathsList.push(fromJS(newPath)));
+      }
+
+      return state;
+    },
+    [REMOVE_PATH]: (state, action) => {
+      const { label } = action;
+      const pathsList = state.get('pathsList');
+      const indexFound = pathsList.findIndex(item => {
+        return item.get('label') === label;
+      });
+
+      if (indexFound >= -1) {
+        return state.set('pathsList', pathsList.splice(indexFound, 1));
+      }
+      return state;
+    },
+    [UPDATE_PATH]: (state, action) => {
+      const newPath = action.path;
+      const pathsList = state.get('pathsList');
+      const indexFound = pathsList.findIndex(item => {
```

Comment:
const indexFound = pathsList.findIndex(item => {
        return item.get('label') === newPath.label;
      });
to
const indexFound = pathsList.findIndex(item => item.get('label') === newPath.label);